doc/changelog.rst

```diff
@@ -1,5 +1,6 @@
 Changelog
 =========
+* :feature:`952` Support API ``get_current_scope()`` and ``ScopeManager.current_scope``
```

Comment:
`Added support for getting the currently active scope (``test``, ``module`` or ``session``) through the new ``get_current_scope`` API. ``session.scopes.current_scope`` is also available`

Reply:
Double-check me on that last one please...

Reply:
The last one should be ``session.scope_manager.current_scope``. I'll fix it.